decam_process.py

```diff
@@ -15,324 +15,378 @@
+        flatBiasIngest(repo, calibrepo, calibdatafiles)
+        defectIngest(repo, calibrepo, defectfiles)
+    elif os.path.exists(os.path.join(calibrepo, 'cpBIAS')):
+        print('Flats and biases were previously ingested, skipping...')
+        defectIngest(repo, calibrepo, defectfiles)
     else:
-        print('Success!')
-        print('Calibrations corresponding to {0} are now ingested in {1}'.format(repo, calibrepo))
+        flatBiasIngest(repo, calibrepo, calibdatafiles)
+        defectIngest(repo, calibrepo, defectfiles)
+
     return
 
 
 def doProcessCcd(repo, calibrepo, processedrepo, visit, ccdnum):
     '''
     Perform ISR with ingested images and calibrations via processCcd
 
-    For successful difference imaging in the next step, astrometry and
-    photometric calibration must also be done by processCcd.
-
-    These steps need a reference catalog. The catalog used here is pan-starrs,
-    which lives on lsst-dev at /datasets/refcats/htm/ps1_pv3_3pi_20170110/
-    This catalog must exist in repo/ref_cats, e.g.,
-    $ ln -s /datasets/refcats/htm/ps1_pv3_3pi_20170110/ /path/to/repo/ref_cats
-    For more information, see RFC-257, DM-8232, and
-    https://community.lsst.org/t/creating-and-using-new-style-reference-catalogs
-
-    USAGE:
-    $ python hits_ingest.py processCcd
+    Parameters
+    ----------
+    repo: `str`
+        The output repository location on disk where ingested raw images live.
+    calibrepo: `str`
+        The output repository location on disk where ingested calibration images live.
+    processedrepo: `str`
+        The output repository location on disk where processed raw images live.
+    visit: `str`
+        One or more DECam visit numbers, e.g., `54321`. Multiple visits may be
+        indicated with standard Butler parsing, e.g., `54321^12345`.
+    ccdnum: `str`
+        One or more DECam CCDs (`1` through `62` are allowed). Setting
+        ccdnum='1..62' will process all of the CCDs.
 
     BASH EQUIVALENT:
     $ processCcd.py repo --id visit=visit ccdnum=ccdnum
             --output processedrepo --calib calibrepo
             -C $OBS_DECAM_DIR/config/processCcdCpIsr.py
             -C processccd_config.py
             --config calibrate.doAstrometry=True calibrate.doPhotoCal=True
-    ** to run from bash, 'processccd_config.py' must exist and contain, e.g.,
-        from lsst.meas.algorithms import LoadIndexedReferenceObjectsTask
-        for refObjLoader in (config.calibrate.astromRefObjLoader,
-                            config.calibrate.photoRefObjLoader,
-                            config.charImage.refObjLoader,
-                            ):
-        refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
-        refObjLoader.ref_dataset_name = 'pan-starrs'
-        refObjLoader.filterMap = {"g": "g",
-                                  "r": "r",
-                                  "VR": "g"}
+    ** to run from bash, 'processccd_config.py' must exist and contain
```

Comment:
What is the equivalent of using the `processccd_config.py` file for this script? Is it hardcoded somewhere, or not used?